examples/nextjs-zklogin/pages/protected.tsx

```diff
@@ -74,7 +74,7 @@ export default withZkLoginSessionRequired(({ session }) => {
           <p>Loading...</p>
         ) : (
           <ul>
-            {txs.txDigests.map((txDigest) => (
+            {txs!.txDigests.map((txDigest) => (
```

Comment:
Any reason we would assert null instead of doing a conditional render such as `txs && ` ?

Reply:
You are right, it's better to check for the error case.